show/muxcable.py

```diff
@@ -0,0 +1,352 @@
+import json
+import sys
+
+import click
+import utilities_common.cli as clicommon
+
+from sonic_py_common import multi_asic, device_info
+from swsscommon import swsscommon
+from swsscommon.swsscommon import SonicV2Connector
```

Comment:
This import seems redundant, as swsscommon is imported on the line above. You can simply reference this by `swsscommon.SonicV2Connector`

Reply:
fixed